### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by ozlemsalehi** [SpinQTech/SpinQKit#1]
After installing spinqkit using the command `pip install spinqkit`, I could not run the provided example in this document [](https://github.com/SpinQTech/SpinQKit/blob/main/doc/GetStarted.md)

I could run it only after downgrading 'ANTLR' and 'python-igraph' libraries to a version that was released around December 2021.

I am using Windows 10 and Python 3.9.

**Comment by yangguohao** [comment]
Thanks for your issues. The spinqkit packages is still in alpha. We have updated the latest version but not push to the repository

**Comment by SpinQTech** [comment]
Hi, 
We just publish a new version which specifies the required packages. SpinQKit is now available on Windows, Linux, and MacOS. Could you please try again and let us know if you see any new issue? Thank you!
